docs/4.x/ab-tests/browser.md

```diff
@@ -107,7 +107,7 @@ _paq.push(['AbTesting::create', {
       }
       return false;
   },
-  matomoTracker: Matomo.getAsyncTracker(piwikUrl, piwikSiteId),
+  piwikTracker: Matomo.getAsyncTracker(matomoUrl, matomoSiteId),
```

Comment:
Guess the `piwikTracker` option hasn't been renamed in the plugin yet?

Reply:
exactly, will change the premium feature docs once we renamed things there